mcc/trek/trek/trek.go

```diff
@@ -9,7 +9,8 @@ import (
 	"regexp"
 )
 
-// RFC1123HostnameRegExp : Hostname Regular expression according with RFC1123HostnameRegExp.
+// RFC1123HostnameRegExp : Hostname Regular expression according with
+// RFC1123HostnameRegExp.
 const RFC1123HostnameRegExp = "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9-]*[a-zA-Z0-9]|\\*)\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9-]*[A-Za-z0-9])$"
```

Comment:
@miguelbernadi I would consider some other way to represent these strings that fits in 80 cols, but that might make code less clear,... do you think it's worht the effort?